test/test_jit.py

```diff
@@ -6457,6 +6457,21 @@ def int2():
 
         self.checkScript(int1, ())	
         self.checkScript(int2, ())
+
+    def test_number_sum(self):	
+        def int1():	
+            return sum(torch.tensor([1,2,3]))
+        def int2():	
+            return sum(torch.tensor([-1,0,1]))	
+        def float1():	
+            return sum(torch.tensor([1.2,2.3,3]))
+        def float2():	
+            return sum(torch.tensor([0.2,0.1,0.2]))
+
+	self.assertEqual(int1(), 6)
```

Comment:
This doesn't use the jit, it's calling the eager mode functions. I think you want `self.checkScript`. There are also mixed tabs/spaces here so I don't think this would run, make sure you have all your tests passing locally before you push/request review. See [CONTRIBUTING.md](https://github.com/pytorch/pytorch/blob/master/CONTRIBUTING.md#unit-testing)